src/dbmanager.cpp

```diff
@@ -57,6 +57,33 @@ bool DBManager::isNoteExist(NoteData* note)
     return query.value(0).toInt() == 1;
 }
 
+NoteData* DBManager::getNote(QString id) {
+    QSqlQuery query;
+
+    int parsedId = id.split('_')[1].toInt();
+    QString queryStr = QStringLiteral("SELECT * FROM active_notes WHERE id = %1 LIMIT 1").arg(parsedId);
+    query.exec(queryStr);
+
+    if (query.first()) {
+        NoteData* note = new NoteData();
```

Comment:
because you're using it in importNote() and importNote() is used in importNotes() which uses QtConcurrent that puts the the imported note in another thread. That's why you get this error.

To use object in another thread it shouldn't have a parent. Well, we'll see what we can do :)